graylog2-web-interface/src/components/common/Select.jsx

```diff
@@ -208,17 +243,23 @@ class Select extends React.Component<Props, State> {
     } = this.props;
 
     return (
-      <div className={`${size === 'small' ? 'select-sm' : ''} ${reactSelectSmStyles.locals.increaseZIndex}`}>
-        <SelectComponent {...rest}
-                         onChange={onReactSelectChange || this._onChange}
-                         isMulti={isMulti}
-                         isDisabled={isDisabled}
-                         getOptionLabel={option => option[displayKey]}
-                         getOptionValue={option => option[valueKey]}
-                         components={_components}
-                         styles={_styles}
-                         value={formattedValue} />
-      </div>
+      <SelectComponent {...rest}
+                       onChange={onReactSelectChange || this._onChange}
+                       isMulti={isMulti}
+                       isDisabled={isDisabled}
+                       getOptionLabel={option => option[displayKey]}
+                       getOptionValue={option => option[valueKey]}
+                       components={{
+                         ..._components,
+                         IndicatorsContainer: isMenuOpen ? IndicatorsContainerOpen : IndicatorsContainerClosed,
```

Comment:
This may be able to be simpler and require less state management.
```
        DropdownIndicator: (base, state) => ({
            ...base,
            transform: state.selectProps.menuIsOpen && 'rotate(180deg)'
        })
```

Reply:
I was looking for something like that, but the `state` object is empty in both the `IndicatorsContainer` and the `DropdownIndicator` component :/

Reply:
Thanks! I was looking for something like that, but looked in the component, which does not receive the state. It is way better now!